modular_skyrat/master_files/code/modules/pain_shock/_defines.dm

```diff
@@ -0,0 +1,9 @@
+#define SHOCK_STAGE_MINOR 20
+#define SHOCK_STAGE_MODERATE 30
+#define SHOCK_STAGE_MAJOR 40
+#define BASE_FLOW_RATE 75
+#define BASE_FLOW_RATE_UPPER 90
+#define FLOW_RATE_ARREST 300
+#define FLOW_RATE_DEAD 0
+#define FLOW_RATE_THREHOLD 50
+#define DAMAGE_TO_PAIN_DIVISION_FACTOR 6
```

Comment:
what?
so if i have a whopping 300 damage i can still walk around with only 50 pain?

Reply:
You'd be somewhere really high